site/documentation/content/user-guide/coap-adapter.md

```diff
@@ -86,15 +86,18 @@ The device is authenticated using PSK.
     delivered to any potential consumer (yet). However, if the message type is `CON` (*at least once* semantics), then the adapter waits
     for the message to be delivered and accepted by a downstream consumer before responding with this status code.
   * 4.00 (Bad Request): The request cannot be processed. Possible reasons include:
-         * the request body is empty and the *URI-query* option doesn't contain the `empty` parameter
+         * the request body is empty, and the *URI-query* option doesn't contain the `empty` parameter.
```

Comment:
no comma here

Reply:
Comma used before `and` linking two independent clauses here.